lib/collections/addon/components/collections-submission/collection-submission-confirmation-modal/template.hbs

```diff
@@ -32,20 +31,20 @@
                 </p>
             {{/if}}
         {{/if}}
-    </modal.body>
-    <modal.footer data-analytics-scope='Collection - Submit'>
+    </dialog.main>
+    <dialog.footer data-analytics-scope='Collection - Submit'>
         <Button
             data-test-collection-submission-confirmation-modal-cancel-button
             data-analytics-name='Cancel submission'
-            {{on 'click' (action modal.close)}}
+            {{on 'click' (action dialog.close)}}
         >
             {{t 'general.cancel'}}
         </Button>
         {{#if @showResubmitModal}}
             <Button
                 data-test-collection-submission-confirmation-modal-resubmit-button
                 data-analytics-name='Confirm submission'
-                {{on 'click' (action modal.submit)}}
+                {{on 'click' (action @resubmitToCollection this.removeReason)}}
```

Comment:
This is much easier to read than having to check what the heck `modal.submit` does depending on whether or not this is a resubmit 👏